src/legacy/core_plugins/kibana/public/discover/np_ready/components/sidebar/_index.scss

```diff
@@ -0,0 +1,217 @@
+.sidebar-container {
```

Comment:
AFAIK the `_index.scss` file should only import local files, not contain rules on its own.

Reply:
Also, it's very important to use plugin specific prefixes, i.e. `dsc` and use [BEM naming](https://eui.elastic.co/pr_3104/#/guidelines/sass).

Reply:
thx, i will have a look, mainly moved css from other sources to this central file

Reply:
seems we have 3 different style naming types of 3 generations of Kibana in one file, will improve that :)

Reply:
so, I've cleaned css up, should be much better now!